app/views/pages/get-started/contact-us-tutorial/building-the-ui.liquid

```diff
@@ -0,0 +1,99 @@
+
+{% include 'alert/tip', content: 'Creating a homepage is a bit special compared to creating a regular [page](/developer-guide/pages/pages). For detailed documentation, refer to the [Homepage](/developer-guide/pages/pages#homepage) documentation.' %}
+
+
+## Requirements
+
+* Followed all steps of the [Quick Setup Guide](/get-started/contact-us-tutorial/#quick-setup-guide) for building 
+* HTML knowledge
+* [Pages](/developer-guide/pages/pages)
+
+## Create a directory for pages
+
+First, create a directory for your pages within the `app` directory:
+
+<pre class="command-line" data-user="user" data-host="host"><code class="language-bash">
+mkdir -p app/views/pages
+</code></pre>
+
+`-p`: an option to be able to create multiple nested directories at once.
+
+{% include 'alert/tip', content: 'You can have multiple files with the same path, so organizing your files clearly is essential.' %}
+
+
+## Create a home page file
+
+We will use HTML and Liquid templating to create the form structure and input fields. Create an `index.html.liquid` file in `app/views/pages/` and add the following HTML code:
+
+#### app/views/pages/index.html.liquid
+
+{% raw %}
+```liquid
+<h2>Contact Us</h2>
+<form>
+    <div>
+        <label for="email">Email</label>
+        <input type="text" name="contact[email]" id="email">
+    </div>
+<div>
+      <textarea name="contact[body]"></textarea>
+    </div>
+    <input type="submit" value="Send">
+  </form>
+```
+{% endraw %}
+
+You might notice that we used `index.html.liquid` for our homepage file instead of `index.html`. This is because our tool can download websites and store them in HTML format, which can still be rendered by platformOS. Although using `.liquid` is not strictly necessary, we recommend it for consistency and ease of use.
+
+{% include 'alert/tip', content: 'The file extension `.html.liquid` specifies HTML as a `format` and Liquid as a render engine, allowing the use of both HTML and Liquid templating. This makes it easy to render dynamic content using platformOS. Read more about [formats](/developer-guide/pages/pages#formats).' %}
+
+## Using slug vs path to the file
+
+While it's possible to add a slug to the page that holds the homepage content, it's not the recommended approach, unless necessary. If someone needs to modify the homepage later, finding the correct file can be challenging if it’s not clearly named. For example, if the homepage content is in a file named `hello.html`, it’s not immediately clear that this file represents the homepage. This can be even more confusing if there are nested directories.
+
+The recommended approach is to use the file path as the default route. This method automatically uses the name of the file as the path, which greatly improves user experience and maintainability. 
+
+The name of the file will be automatically used as the path. In our case, `app/views/pages/index.html.liquid` will default to `/`.
```

Comment:
I would recommend using the 'good' approach as a default and then just saying that more is possible with the link to the docs as currently I am starting to learn what should I not do and just after what I need to do.

Reply:
Rewrote that part.